### PR TITLE
changelog v6.5.0-beta.2

### PR DESCRIPTION
**Pull request by RingerJK**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ Mapbox welcomes participation and contributions from everyone.
 
 ### main
 
+### v6.5.0-beta.2 - April 14, 2022
+- Fixed exceptions for `waypointName` item has `&` char in a string. [#1410](https://github.com/mapbox/mapbox-java/pull/1410)
+
+#### Known issues
+- `RouteOptions` crashes if `waypointsName` item contains `&` char at the start or at the end of a string.
```
Comment (LukasPaczos) [inline CHANGELOG.md:11]:
```suggestion
- `RouteOptions#toUrl` might produce invalid URLs (and fail a request or later crash on `RouteOptions#fromUrl`) if query parameters contain `&` chars that do not exactly match the `<white space>&<white space>` pattern. Only this specific pattern is escaped correctly.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ Mapbox welcomes participation and contributions from everyone.
 
 ### main
 
+### v6.5.0-beta.2 - April 14, 2022
+- Fixed exceptions for `waypointName` item has `&` char in a string. [#1410](https://github.com/mapbox/mapbox-java/pull/1410)
```
Comment (LukasPaczos) [inline CHANGELOG.md:8]:
```suggestion
- Added correct escaping for `<white space>&<white space>` pattern in query parameters for `RouteOptions#toUrl`. [#1410](https://github.com/mapbox/mapbox-java/pull/1410)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,12 @@ Mapbox welcomes participation and contributions from everyone.
...
+
+#### Known issues
+- `RouteOptions#toUrl` might produce invalid URLs (and fail a request or later crash on `RouteOptions#fromUrl`) if query parameters contain `&` chars that do not exactly match the `<white space>&<white space>` pattern. Only this specific pattern is escaped correctly.
```
Comment (LukasPaczos) [inline CHANGELOG.md:11]:
```suggestion
- `RouteOptions#toUrl` might still produce invalid URLs (and fail a request or later crash on `RouteOptions#fromUrl`) if query parameters contain `&` chars that do not exactly match the `<white space>&<white space>` pattern. Only this specific pattern is escaped correctly.
```

**Review by LukasPaczos — COMMENTED** [review]
Sorry, one more.

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,12 @@ Mapbox welcomes participation and contributions from everyone.
 
 ### main
 
+### v6.5.0-beta.2 - April 14, 2022
+- Added correct escaping for `<white space>&<white space>` pattern in query parameters for `RouteOptions#toUrl`. [#1410](https://github.com/mapbox/mapbox-java/pull/1410). [#1410](https://github.com/mapbox/mapbox-java/pull/1410)
```
Comment (VysotskiVadim) [inline CHANGELOG.md:8]:
```suggestion
- Added correct escaping for `<white space>&<white space>` pattern in query parameters for `RouteOptions#toUrl`. [#1410](https://github.com/mapbox/mapbox-java/pull/1410).
```

Comment (RingerJK) [reply CHANGELOG.md:8]: oh, nice shot 👍

**Review by VysotskiVadim — APPROVED** [review]

**Review by LukasPaczos — APPROVED** [review]

**Pull request merged**